include/date_time.h

```diff
@@ -129,12 +129,6 @@ void date_time_register_handler(date_time_evt_handler_t evt_handler);
  */
 int date_time_update_async(date_time_evt_handler_t evt_handler);
 
-/** @brief Clear the current date time held by the library.
- *
- *  @return 0 If the operation was successful.
- */
-int date_time_clear(void);
```

Comment:
Reminder to add release note entry for removal of this function

Reply:
I just recalled. `date_time_clear()` is used in:

https://github.com/nrfconnect/sdk-nrf/blob/cfedfdfa08567b2252b511a4d1db15fbeba8152d/tests/lib/date_time/src/main.c#L275

So you'll have to find a way to circumvent that.

Reply:
This removal has been reverted.